pkgs/development/compilers/bambu/default.nix

```diff
@@ -0,0 +1,257 @@
+{ runCommand
+, makeWrapper
+, buildEnv
+, fetchFromGitHub
+, stdenv
+, lib
+, pkgsi686Linux
+, libmpc
+, gmp
+, mpfr
+, boost
+, libxml2
+, xz
+, zlib
+, icu
+, suitesparse
+, mpfi
+, glpk
+, autoconf
+, autoconf-archive
+, automake
+, libtool
+, bison
+, doxygen
+, flex
+, graphviz
+, verilog
+, verilator
+, gmpxx
+, buddy
+, bash
+  # 32bit Clang with a supported major version
+, clangFrontends ? [ pkgsi686Linux.llvmPackages_7 pkgsi686Linux.llvmPackages_8 pkgsi686Linux.llvmPackages_10 pkgsi686Linux.llvmPackages_11 pkgsi686Linux.llvmPackages_12 pkgsi686Linux.llvmPackages_13 pkgsi686Linux.llvmPackages_16 ]
+  # 32bit GCC with a supported major version
+, gccFrontends ? [ pkgsi686Linux.gcc8 pkgsi686Linux.gcc6 pkgsi686Linux.gcc7 ]
+}:
+let
+  # The clang 9 frontend is currently broken in bambu
+  supportedLlvmMajorVersions = [ "7" "8" "10" "11" "12" "13" "16" ];
+  supportedGccMajorVersions = [ "6" "7" "8" ];
+
+  wrapGcc = gcc: runCommand (gcc.name + "-with-libs")
+    {
+      buildInputs = [ makeWrapper ];
```

Comment:
build time tools must be in `nativeBuildInputs` and `makeWrapper` should always be in `nativeBuildInputs`